### PR TITLE
Adding in some accessibility-related language

### LINKED ISSUE #860: Add accessibility section
In the [WebApps charter](https://www.w3.org/2019/05/webapps-charter.html#scope) it says:
>Where there are implications for implementors, developers, or users, in the areas of accessibility, internationalization, privacy, and security, each specification must have a section that describes relevant benefits, limitations, and best practice solutions for that particular area.>

To this end each spec needs to have an accessibility section. For the avoidance of doubt in the reader, the section should be included even if it indicates there are no known accessibility considerations.

Relates to w3c/webappswg#15

### PR DESCRIPTION
**Pull request by aarongustafson**
Closes #860

This change (choose one):

* [ ] Breaks existing normative behavior (please add label "breaking")
* [ ] Adds new normative requirements
* [x] Adds new normative recommendations or optional items
* [x] Makes only editorial changes (only changes informative sections, or
  changes normative sections without changing behavior)
* [ ] Is a "chore" (metadata, formatting, fixing warnings, etc).

Commit message:

Taking a crack at adding accessibility info. The only outstanding item is adding a recommendation for screenshot `label`s once ImageResource is integrated.

***
<a href="https://pr-preview.s3.amazonaws.com/aarongustafson/manifest/pull/898.html" title="Last updated on Jun 30, 2020, 12:35 AM UTC (51335ba)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/manifest/898/6287a3a...aarongustafson:51335ba.html" title="Last updated on Jun 30, 2020, 12:35 AM UTC (51335ba)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4093,6 +4124,27 @@ <h3>
...
+        </li>
+		<li>[[MEDIAQUERIES-5]] defines the following terms:
+          <ul>
+            <li>
+              <a data-cite="MEDIAQUERIES-5#prefers-color-scheme"><dfn>user-preferred
+	          color scheme</dfn></a>
+            </li>
+          </ul>
+        </li>
+        mediaqueries-5
```
Comment (marcoscaceres) [inline index.html:4147]: We can get these exported from the appropriate spec...  

```suggestion
```

File: index.html [hunk 2]
```diff
@@ -2174,6 +2189,14 @@ <h3>
           <li>Return <var>color</var>.
           </li>
         </ol>
+		<section>
+          <h4>Accessible Colors</h3>
+          <p>
+            Implementors MAY choose to override the value defined by the
```
Comment (marcoscaceres) [inline index.html:2195]:
```suggestion
            Implementors MAY override the value defined by the
```

File: index.html [hunk 3]
```diff
@@ -1779,6 +1779,13 @@ <h3>
           (e.g., amongst a list of other applications, or as a label for an
           icon).
         </p>
+		<section>
+          <h4>Accessible Name</h3>
+          <p>
+            The [=accessible name=] of the web application is equal to the
+            value of the <code>name</code> member.
```
Comment (marcoscaceres) [inline index.html:1786]:
```suggestion
            value of the {{WebAppManifest/name}} member.
```

File: index.html [hunk 4]
```diff
@@ -1779,6 +1779,13 @@ <h3>
           (e.g., amongst a list of other applications, or as a label for an
           icon).
         </p>
+		<section>
```
Comment (marcoscaceres) [inline index.html:1782]: Not sure we need an entirely new section for this... let's just move the `<p>` into the the description of the member.  Same for the other ones.

File: index.html [hunk 5]
```diff
@@ -1799,6 +1806,14 @@ <h3>
           The <dfn>description</dfn> member allows the developer to describe
           the purpose of the web application.
         </p>
+		<section>
+          <h4>Accessible Description</h3>
+          <p>
+            The [=accessible description=] of the web application is equal to 
+			the value of the <code>description</code> member, if it is provided
```
Comment (marcoscaceres) [inline index.html:1813]:
```suggestion
			the value of the {{WebAppManifest/description}} member, if it is provided
```

File: index.html [hunk 6]
```diff
@@ -2293,6 +2316,14 @@ <h3>
           <li>Return <var>color</var>.
           </li>
         </ol>
+		<section>
+          <h4>Accessible Colors</h3>
+          <p>
+            Implementors MAY choose to override the value defined by the
```
Comment (marcoscaceres) [inline index.html:2322]:
```suggestion
            Implementors MAY override the value defined by the
```

File: index.html [hunk 7]
```diff
@@ -2293,6 +2316,14 @@ <h3>
...
+            <a>background_color</a> member to support a [=user-preferred color
```
Comment (marcoscaceres) [inline index.html:2323]:
```suggestion
            {{WebAppManifest/background_color}} member to support a [=user-preferred color 
```

**Comment by marcoscaceres** [comment]
Lookin good, but could also use a quick tidy 🙏

**Comment by aarongustafson** [comment]
@marcoscaceres I merged in the removed color processing steps too.

File: index.html [hunk 8]
```diff
@@ -1433,6 +1433,10 @@ <h3>
           (e.g., amongst a list of other applications, or as a label for an
           icon).
         </p>
+        <p>
+          The [=accessible name=] of the web application is equal to the value
```
Comment (marcoscaceres) [inline index.html:1437]: Maybe we should phrase this as:

> The {{WebAppManifest/name}} member serves as the [=accessible name=] of an [=installed web application=]. 

@aarongustafson wdyt? ... same with accessible description.

**Comment by marcoscaceres** [comment]
Blocked on getting the A11y folk to export their definitions...

**Comment by aarongustafson** [comment]
Did I do that right, @marcoscaceres?

```html
        <p>
          The {{WebAppManifest/name}} member serves as the <a data-cite=
          "accname#dfn-accessible-name">accessible name</a> of an [=installed
          web application=].
        </p>
```

**Comment by aarongustafson** [comment]
@marcoscaceres I went old school on the definitions here to unblock merging this PR. Let me know if I missed anything. Respec isn’t complaining so it may be a good stopgap.

File: index.html [hunk 9]
```diff
@@ -1426,6 +1426,10 @@ <h3>
           (e.g., amongst a list of other applications, or as a label for an
           icon).
         </p>
+        <p>
+          The {{WebAppManifest/name}} member serves as the <a>Accessible
```
Comment (marcoscaceres) [inline index.html:1430]:
```suggestion
          The {{WebAppManifest/name}} member serves as the <a>accessible
```

File: index.html [hunk 10]
```diff
@@ -1426,6 +1426,10 @@ <h3>
...
+          Name</a> of an [=installed web application=].
```
Comment (marcoscaceres) [inline index.html:1431]:
```suggestion
          name</a> of an [=installed web application=].
```

File: index.html [hunk 11]
```diff
@@ -3374,6 +3393,22 @@ <h2>
             </li>
           </ul>
         </li>
+        <li>
+          <a href="https://w3c.github.io/accname">Accessible Name and
+          Description Computation</a> defines the following terms:
+          <ul>
+            <li>
+              <a href=
+              "https://w3c.github.io/accname/#dfn-accessible-name"><dfn data-dfn-for="">
```
Comment (marcoscaceres) [inline index.html:3402]:
```suggestion
              "https://w3c.github.io/accname/#dfn-accessible-name"><dfn>
```

File: index.html [hunk 12]
```diff
@@ -3374,6 +3393,22 @@ <h2>
...
+              accessible name</dfn></a>
+            </li>
+            <li>
+              <a href=
+              "https://w3c.github.io/accname/#dfn-accessible-description"><dfn data-dfn-for="">
```
Comment (marcoscaceres) [inline index.html:3407]:
```suggestion
              "https://w3c.github.io/accname/#dfn-accessible-description"><dfn>
```

**Pull request merged**